systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java

```diff
@@ -4,133 +4,55 @@
+    @Override
+    public void create(KafkaConnect resource) {
+        kafkaConnectClient().inNamespace(resource.getMetadata().getNamespace()).withName(resource.getMetadata().getName()).createOrReplace(resource);
     }
-
-    public static KafkaConnectBuilder kafkaConnectWithMetrics(String name, int kafkaConnectReplicas) {
-        return kafkaConnectWithMetrics(name, name, kafkaConnectReplicas);
+    @Override
+    public void delete(KafkaConnect resource) throws Exception {
+        kafkaConnectClient().inNamespace(resource.getMetadata().getNamespace()).withName(
+            resource.getMetadata().getName()).withPropagationPolicy(DeletionPropagation.FOREGROUND).delete();
     }
-
-    public static KafkaConnectBuilder kafkaConnectWithMetrics(String name, String clusterName, int kafkaConnectReplicas) {
-        KafkaConnect kafkaConnect = getKafkaConnectFromYaml(Constants.PATH_TO_KAFKA_CONNECT_METRICS_CONFIG);
-
-        ConfigMap metricsCm = TestUtils.configMapFromYaml(Constants.PATH_TO_KAFKA_CONNECT_METRICS_CONFIG, "connect-metrics");
-        KubeClusterResource.kubeClient().getClient().configMaps().inNamespace(kubeClient().getNamespace()).createOrReplace(metricsCm);
-
-        return defaultKafkaConnect(kafkaConnect, name, clusterName, kafkaConnectReplicas);
-    }
-
-    private static KafkaConnectBuilder defaultKafkaConnect(KafkaConnect kafkaConnect, String name, String kafkaClusterName, int kafkaConnectReplicas) {
-        return new KafkaConnectBuilder(kafkaConnect)
-            .withNewMetadata()
-                .withName(name)
-                .withNamespace(ResourceManager.kubeClient().getNamespace())
-                .withClusterName(kafkaClusterName)
-            .endMetadata()
-            .editOrNewSpec()
-                .withVersion(Environment.ST_KAFKA_VERSION)
-                .withBootstrapServers(KafkaResources.tlsBootstrapAddress(kafkaClusterName))
-                .withReplicas(kafkaConnectReplicas)
-                .withNewTls()
-                    .withTrustedCertificates(new CertSecretSourceBuilder().withNewSecretName(kafkaClusterName + "-cluster-ca-cert").withCertificate("ca.crt").build())
-                .endTls()
-                .addToConfig("group.id", KafkaConnectResources.deploymentName(kafkaClusterName))
-                .addToConfig("offset.storage.topic", KafkaConnectResources.configStorageTopicOffsets(kafkaClusterName))
-                .addToConfig("config.storage.topic", KafkaConnectResources.metricsAndLogConfigMapName(kafkaClusterName))
-                .addToConfig("status.storage.topic", KafkaConnectResources.configStorageTopicStatus(kafkaClusterName))
-                .withNewInlineLogging()
-                    .addToLoggers("connect.root.logger.level", Environment.STRIMZI_COMPONENTS_LOG_LEVEL)
-                .endInlineLogging()
-            .endSpec();
+    @Override
+    public boolean isReady(KafkaConnect resource) {
```

Comment:
Same as the other comments. I will not mention it anymore, but it should be fixed everywhere.